src/components/menus/TertiaryNav/examples/TertiaryNavWithDrawerExample.tsx

```diff
@@ -0,0 +1,90 @@
+/* @ts-nocheck */
+import Drawer from '../../Drawer/Drawer';
+import { PrimaryButton } from '../../../buttons/PrimaryButton/PrimaryButton';
+import { TertiaryNav, TertiaryNavItem } from '../TertiaryNav';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Item2 = () => (<div>Item 2 Content</div>);
+const Item3 = () => (<div>Item 3 Content</div>);
+
+interface IProps {
+	onChange: (target: any) => undefined;
+}
+
+class Item1 extends React.Component<IProps, { value: string }> {
+	constructor (props: IProps) {
+		super(props);
+
+		this.state = {
+			value: 'change this text',
+		};
+
+		this.handleChange = this.handleChange.bind(this);
```

Comment:
Not at all a showstopper in my book, but it would be nice to use arrow funcs and avoid calls to `bind`